dts/common/nordic/nrf54l15.dtsi

```diff
@@ -212,6 +212,7 @@
 				ieee802154-supported;
 				ble-2mbps-supported;
 				ble-coded-phy-supported;
+				cs-supported;
```

Comment:
You might need to delete this property for nrf54l15bsim.
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/native/nrf_bsim/nrf54l15bsim_nrf54l15_cpuapp.dts#L101